lib/elixir_newbie_web/live/blog_show.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ElixirNewbieWeb.BlogShow do
+  use ElixirNewbieWeb, :live_view
+  alias ElixirNewbie.Blog
+  alias ElixirNewbie.Stats
+  alias ElixirNewbie.Blog.Post
+  ## alias ElixirNewbie.Router.Helpers, as: Routes ##
+
+
+  def render(assigns) do ## BigSpaces Wonders... about embedding the  {/if}  <figure class="makeup-blog"> as in v1
```

Comment:
We could only use that syntax because we used Surface UI in the old project. Now we use embedded Elixir.

```elixir
<%= if condition do %>
some html
<% else %>
some other html
<% end %>

<% for n <- list do %>
some html
<% end %>
```

Reply:
Thanks for clarifying!